ScoutSuite/frontend/src/api/useResources.js

```diff
@@ -0,0 +1,38 @@
+import { useEffect, useState } from 'react';
+import * as API from './api';
+
+/**
+ * React Hook to fetch API data and re-render the component
+ * @param {*} path
+ */
+export const useResources = (service, resource, ids) => {
+  const [data, setData] = useState([]);
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    const asyncAPI = async () => {
+      console.log('GET RESOURCES', service, resource, ids);
+      if (service && resource && ids) {
+        setLoading(true);
+        try {
+          const requests = ids.map(id =>
+            API.get(`services/${service}/resources/${resource}/${id}`),
+          );
+          console.log('list de requests', requests);
```

Comment:
dont forget to remove the console.logs once the draft PR is done.

Reply:
Fixed